pkgs/development/compilers/llvm/common/llvm/default.nix

```diff
@@ -135,10 +128,23 @@ in stdenv.mkDerivation (rec {
     substituteInPlace unittests/IR/CMakeLists.txt \
       --replace "PassBuilderCallbacksTest.cpp" ""
     rm unittests/IR/PassBuilderCallbacksTest.cpp
+  '' + (lib.optionalString (lib.versionAtLeast release_version "13") ''
```

Comment:
As far as I know, you don't need the extra set of parens here (or below)

Reply:
Without this blank line, it actually causes a rebuild. We don't want rebuilds.

Reply:
Removed the extra parentheses.